### ISSUE TITLE
Wrong image order when running on a dir using console interface

### ISSUE THREAD

**Issue by BostonLobster** [OPHoperHPO/image-background-remove-tool#80]
I use the following command to run default carvekit on a dir:
```
python -m carvekit -i ./images -o ./masked_images --device cuda
```

However, when I check the results, the output images are out-of-order.

For example, the `images/001.jpg` is
![001](https://user-images.githubusercontent.com/47711465/174438927-92e07e97-22ab-4782-8e66-662af2ac474f.jpg)

but the result is
![001](https://user-images.githubusercontent.com/47711465/174438938-7ad09877-9396-4acc-92b7-a5ecd5b7996a.png)

which corresponds to another input image.

Very weird thing, could you please check this problem?

**Comment by Larry-u** [comment]
Same issue here

**Comment by OPHoperHPO** [comment]
Hello, @BostonLobster. There was a typo in the CLI code that caused this issue. This has been fixed by @Larry-u in PR #81.
